dist/angular-validation.js

```diff
@@ -270,6 +270,28 @@
     };
 
     /**
+     * Add Message Element at the initial phase
+     * Do this function when to custom your messageElement 
+     * NODE: this funtion have similar purpose with `message-id` attribute. 
+     * This function will help you add your `messageElement` automatically instead of pre-defined.
+     * @param element
+     */
+    this.addMsgElement = function(element) {
+      return element.after('<span></span>');
```

Comment:
I get your point, using `add` is more understandable in this state - adding a non-existing item.
Then let's stick on the original one `add` :+1: